src/main/java/org/broadinstitute/hellbender/utils/test/IntegrationTestSpec.java

```diff
@@ -94,17 +92,20 @@ public void executeTest(final String name, CommandLineProgramTest test) throws I
             tmpFiles.add(fl);
         }
 
-        final String args = String.format(getArgs(), tmpFiles.toArray());
+        final String preFormattedArgs = getArgs();
+        final String formattedArgs = String.format(preFormattedArgs, tmpFiles.toArray());
```

Comment:
Won't this line throw an `IllegalFormatException` if the spec is missing a placeholder, meaning that we'll never reach your new user-friendly error below?